model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/eq/AddressByEQPostcodeResponse.scala

```diff
@@ -19,14 +20,16 @@ import uk.gov.ons.addressIndex.model.server.response.address.AddressResponseAddr
   */
 case class AddressByEQPostcodeResponse(postcode: String,
                                        addresses: Seq[AddressResponseAddressPostcodeEQ],
+                                       postcodes: Option[Seq[AddressResponsePostcodeGroup]],
```

Comment:
I would tend to favour just the Seq here as its either empty or not. Similar to option.

Reply:
It needs to be an option so that the JSON response where the combo is not selected is unchanged. Otherwise all the "regular" responses will contain _postcodes: []_ which is at best untidy and at worst a breaking change (if the UI is not optimally coded).